src/fabric/NativeScreensModule.web.ts

```diff
@@ -0,0 +1,22 @@
+import type { Double, Int32 } from 'react-native/Libraries/Types/CodegenTypes';
+
+export default {
+  startTransition(_tag: Int32 | null) {
+    console.log(
+      '[ReactNativeScreens] startTransition is not supported on web]'
```

Comment:
Let's stick to the one pattern:

```suggestion
      '[RNScreens] startTransition is not supported on web.'
```